Discover/Views/DiscoverViewController.swift

```diff
@@ -0,0 +1,276 @@
+//
+//  DiscoverViewController.swift
+//  Discover
+//
+//  Created by Егор Бадмаев on 27.10.2022.
+//  
+//
+
+import UIKit
+import Resources
+import Models
+import CommonUI
+
+final class DiscoverViewController: UIViewController {
+    
+    // MARK: - Private Properties
+    
+    private let output: DiscoverViewOutput
+    /// Link to the next page.
+    private var nextPageUrl: String?
+    /// Array of recipes.
+    private var data: [Recipe] = []
+    /// Defines whether fetching is in progress. It is being used for pagination.
+    private var isFetchingInProgress = false
+    
+    /// Refresh control to implement _pull to refresh_
+    private lazy var refreshControl: UIRefreshControl = {
+        let refreshControl = UIRefreshControl()
+        refreshControl.addTarget(self, action: #selector(handleRefreshControl), for: .valueChanged)
+        return refreshControl
+    }()
```

Comment:
Change color of the refresh control (because on some iOS version it is gray, on others it is black)